tests/integration/create-switch-account.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserDriver, createTestSelector, setupBrowser } from './utils';
+import { WalletPage } from './page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import { SettingsSelectors } from './settings.selectors';
+
+jest.setTimeout(40_000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+describe(`Create and switch account`, () => {
+  const COUNT = 3;
+  let browser: BrowserDriver;
+  let wallet: WalletPage;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    wallet = await WalletPage.init(browser, ScreenPaths.INSTALLED);
+  }, 10000);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('should be able to create a new account then switch', async () => {
+    await wallet.signUp();
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector('settings-create-an-account'));
+    await wallet.page.waitForSelector(createTestSelector('create-account-done-button'))
+    await wallet.page.click(createTestSelector('create-account-done-button'));
```

Comment:
Would be great to move these selectors to the page object so we 're not repeating the strings

Reply:
Fixed